Apps/Sandcastle/gallery/Picking.html

```diff
@@ -144,6 +144,50 @@
     }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
 });
 
+Sandcastle.addToolbarButton('Pick position', function() {
```

Comment:
Any idea why we get negative altitudes right at the end of the model?

![image](https://cloud.githubusercontent.com/assets/782098/7847821/429a0418-0494-11e5-99b6-c294ef826c3f.png)